app/models/claim.rb

```diff
@@ -19,4 +19,11 @@ def self.fetch_all(headers)
       end
     end.flatten
   end
+
+  def self.upload_document(headers, file_name, file_body, claim_id, tracked_item_id)
```

Comment:
Probably we will need some sort of security model that associates a claim with a user so that you can't just look at random claim IDs if they don't belong to you.